components/dashboard/src/components/modal.css

```diff
@@ -4,6 +4,10 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+@tailwind base;
```

Comment:
I assume we need to have this at the beginning of any css file where we're using a tailwind directive now?

Reply:
supposedly it [shouldn't be needed with the `postcss-import` plugin](https://tailwindcss.com/docs/adding-custom-styles#using-multiple-css-files) in the `postcss.config.js` but I spend a lot of time yesterday trying to make it work but couldn't :disappointed: 
because it was just three files for now I just copy the directive, but it would be good to fix it later